docs/ancestry.md

```diff
@@ -730,14 +739,13 @@ ts = msprime.sim_ancestry(
 ts
 ```
 
-To isolate tree sequences for each chromosome, we can trim the full tree sequence
-for each chromosome by defining the end points of each chromosome and using
-the {meth}`~tskit.TreeSequence.keep_intervals` method.
-It is important to specify `simplify=False` so
-that node indices remain consistent across chromosomes. The
-{meth}`~tskit.TreeSequence.trim` method
-is then used to trim the tree sequences to the focal chromosome, so that position
-indices begin at 0 within each chromosome tree sequence.
+To isolate tree sequences for each chromosome, we can trim the full tree
```

Comment:
Maybe "To put each chromosome in its own separate tree sequence"? At first when I read "isolate" I thought that we were just going to cut out the inter-chromosome spacing (ie mark the data as missing there) but keep it all in one tree sequence.